lib/src/components/NxProgressBar/NxProgressBar.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { faExclamationCircle, faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+import classnames from 'classnames';
+import React, { forwardRef } from 'react';
+
+import { useUniqueId } from '../../util/idUtil';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, propTypes } from './types';
+
+import './NxProgressBar.scss';
+
+const NxProgressBar = forwardRef<HTMLProgressElement, Props>(
+    function NxProgressBar(props, ref) {
+      const {
+        hasError,
+        inlineCounter = false,
+        label,
+        labelError,
+        labelSuccess,
+        max: maxProp,
+        showCounter = true,
+        value,
+        variant
+      } = props;
+
+      const max = maxProp || 100;
+      const percentage = Math.round(value / max * 100);
+
+      const progressId = useUniqueId('nx-progress-bar__progress');
+
+      const renderCounter = () => {
+        if (!showCounter || variant === 'inline') {
+          return null;
+        }
+
+        return (
+          <div className="nx-progress-bar__counter">
```

Comment:
What's the purpose of this wrapper element?

Reply:
> What's the purpose of this wrapper element?

Removed: e7b1944aa769a038f569858f8b2f13f52bca69aa